docs/UserGuide.md

```diff
@@ -87,6 +87,19 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
+### 2. Adds a nurse to the Healthcare Xpress book
+
+Format:`add c/N n/NAME p/PHONE_NUMBER e/EMAIL g/GENDER [t/TAG]`
+
+> 💡 Tip: A nurse can have any number of tags (including 0).
+> You may type it in any order.
+> To add a nurse, type c/N specifically.
```

Comment:
Maybe for here can make it as point form and list it down instead of 3 tips is in 1 continuous line.